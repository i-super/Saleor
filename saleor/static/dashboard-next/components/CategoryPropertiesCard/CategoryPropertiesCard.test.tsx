import * as React from "react";
import { MemoryRouter } from "react-router-dom";
import * as renderer from "react-test-renderer";

import CategoryPropertiesCard from "./CategoryPropertiesCard";

describe("<CategoryPropertiesCard />", () => {
  it("renders while data is loading", () => {
    const component = renderer.create(
      <MemoryRouter>
        <CategoryPropertiesCard description="" onDelete={jest.fn()} title="" />
      </MemoryRouter>
    );
    expect(component).toMatchSnapshot();
  });
  it("renders when data is loaded", () => {
    const component = renderer.create(
      <MemoryRouter>
        <CategoryPropertiesCard
          description="These are the best shoes."
          onDelete={jest.fn()}
          onEdit={jest.fn()}
          title="Shoes"
        />
      </MemoryRouter>
    );
    expect(component).toMatchSnapshot();
  });
});
