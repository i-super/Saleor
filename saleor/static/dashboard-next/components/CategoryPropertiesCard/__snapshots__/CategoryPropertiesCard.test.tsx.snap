// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CategoryPropertiesCard /> renders when data is loaded 1`] = `
<div
  className="MuiPaper-root-2 MuiPaper-elevation2-6 MuiPaper-rounded-3 MuiCard-root-1"
>
  <div
    className="Component-root-32"
  >
    <h2
      className="MuiTypography-root-35 MuiTypography-title-41 Component-title-34"
    >
      Shoes
    </h2>
    <div
      className="Component-action-29"
    >
      <button
        className="MuiButtonBase-root-65 MuiIconButton-root-59"
        disabled={false}
        onBlur={[Function]}
        onClick={[MockFunction]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex="0"
        type="button"
      >
        <span
          className="MuiIconButton-label-64"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-67"
            color={undefined}
            focusable="false"
            viewBox="0 0 24 24"
          >
            <g>
              <path
                d="M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM19 4h-3.5l-1-1h-5l-1 1H5v2h14V4z"
              />
            </g>
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root-73"
        />
      </button>
      <button
        className="MuiButtonBase-root-65 MuiIconButton-root-59"
        disabled={false}
        onBlur={[Function]}
        onClick={[MockFunction]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex="0"
        type="button"
      >
        <span
          className="MuiIconButton-label-64"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-67"
            color={undefined}
            focusable="false"
            viewBox="0 0 24 24"
          >
            <g>
              <path
                d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34c-.39-.39-1.02-.39-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
              />
            </g>
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root-73"
        />
      </button>
    </div>
  </div>
  <div
    className="MuiCardContent-root-80"
  >
    <p
      className="MuiTypography-root-35 MuiTypography-body1-44"
    >
      These are the best shoes.
    </p>
  </div>
</div>
`;

exports[`<CategoryPropertiesCard /> renders while data is loading 1`] = `
<div
  className="MuiPaper-root-2 MuiPaper-elevation2-6 MuiPaper-rounded-3 MuiCard-root-1"
>
  <div
    className="Component-root-32"
  >
    <h2
      className="MuiTypography-root-35 MuiTypography-title-41 Component-title-34"
    >
      
    </h2>
    <div
      className="Component-action-29"
    >
      <button
        className="MuiButtonBase-root-65 MuiIconButton-root-59"
        disabled={false}
        onBlur={[Function]}
        onClick={[MockFunction]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex="0"
        type="button"
      >
        <span
          className="MuiIconButton-label-64"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-67"
            color={undefined}
            focusable="false"
            viewBox="0 0 24 24"
          >
            <g>
              <path
                d="M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM19 4h-3.5l-1-1h-5l-1 1H5v2h14V4z"
              />
            </g>
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root-73"
        />
      </button>
      <button
        className="MuiButtonBase-root-65 MuiIconButton-root-59"
        disabled={false}
        onBlur={[Function]}
        onClick={undefined}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex="0"
        type="button"
      >
        <span
          className="MuiIconButton-label-64"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-67"
            color={undefined}
            focusable="false"
            viewBox="0 0 24 24"
          >
            <g>
              <path
                d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34c-.39-.39-1.02-.39-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
              />
            </g>
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root-73"
        />
      </button>
    </div>
  </div>
  <div
    className="MuiCardContent-root-80"
  >
    <p
      className="MuiTypography-root-35 MuiTypography-body1-44"
    >
      
    </p>
  </div>
</div>
`;
